src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -80,45 +59,72 @@ public Task<IEnumerable<TContext>> Run(TimeSpan? testExecutionTimeout = null)
 
             Console.WriteLine("Total time for testrun: {0}", sw.Elapsed);
 
-            return runDescriptors.Select(r => (TContext)r.ScenarioContext);
+            return runDescriptors.Select(r => (TContext) r.ScenarioContext);
```

Comment:
This does not prevent this PR from being merged, or does it ;)?

On Sunday, February 28, 2016, Andreas Öhlund <notifications@github.com>
wrote:

> In src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs
> <https://github.com/Particular/NServiceBus/pull/3514#discussion_r54349159>
> :
>
> > @@ -80,45 +59,72 @@ public Task<IEnumerable<TContext>> Run(TimeSpan? testExecutionTimeout = null)
> >
> >              Console.WriteLine("Total time for testrun: {0}", sw.Elapsed);
> >
> > -            return runDescriptors.Select(r => (TContext)r.ScenarioContext);
> > +            return runDescriptors.Select(r => (TContext) r.ScenarioContext);
>
> [image: :+1:] to change the template, pull on repo standards?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3514/files#r54349159>.
>


-- 
*Don't miss the Async/Await Webinar Series*
*• Learn more <http://go.particular.net/l/27442/2016-02-15/681bjl> →*